packages/collections/persistent/persistent.pony

```diff
@@ -0,0 +1,7 @@
+"""
+# Persistent Collections Package
+
+List - A persistent list with functional transformations.
+
+Map - A persistent map based on Bagwell's hash array mapped trie algorithm.
+"""
```

Comment:
GitHub doesn't show it as an empty line at the end of the file - it just shows a red "error" symbol at the end of the text if you don't have it.  It looks like now you added two empty lines, to make one show up on GitHub, but this is one too many :stuck_out_tongue: